[DRAFT] Rust Foundation: December 2024 Update (#13)
JoelMarcey/alpha-omega: alpha/engagements/2024/Rust Foundation/update-2024-12.md

> December is usually a bit of a slower month at the Foundation, with many taking longer holiday breaks, etc. That was the case this month, but not without some good updates around crate signing, crates.io improvements and CI cost reduction.

Reviewer: ```suggestion
December is usually a bit of a slower month at the Foundation, with many taking longer holiday breaks, etc. That was the case this month, but there were some good updates around crate signing, crates.io improvements and CI cost reduction.
```

The double-negative tripped up my brain a bit
